builder/draft-claise-yang-full-include.xml

```diff
@@ -85,109 +116,161 @@
+                <li> A module MUST NOT use the "full:include" statement with its own prefix as argument. This rule prevents any infinite recursion in the mounted schemas.</li>
+                <li> Modules used as arguments for the "full:include" statement MUST be valid and compile successfully, independently of the module it is used in.</li>
             </ul>
         </section>
-</section>
-<section anchor="full-include-of-YANG-schema-nodes">
-    <name>Full Include of YANG Schema Nodes</name>
-    <section>
-        <name>Included Data Nodes Path Adjustment</name>
+    </section>
+
+    <!-- [We’re not yet there, let’s keep it simple for now]section>
+        <name>Implementation strategies</name>
+        <section>
+            <name>Re-using YANG schema mount</name>
+        </section>
+        <section>
+            <section anchor="full-include-of-YANG-schema-nodes">
+                <name>Defining a new extension</name>
+                <section>
+                    <name>Included Data Nodes Path Adjustment</name>
+                    <t>
+                        In the case of the call to a "leafref" statement, one must be able to specify the path of a leaf, even through a module passed as an argument to a "full-include" statement.
+                        The paths of the nodes contained in the module (referred in the following paragraph as "the included nodes") used in a "full-include" statement are computed as follows.
+                        TODO: explain how the path are computed, which basically boils down to concatenating the main module path with the included one ?
+                    </t>
+                </section>
+                <section anchor="rpc-operations">
+                    <name>RPC operations</name>
+                    <t>
+                        Todo, read the schema-mount RFC part on this, and apply to our case
+                    </t>
+                </section>
+            </section>
+        </section>
+    </section-->
+
+    <section anchor="yang-module-code">
+        <name>ietf-full-include YANG module</name>
         <t>
-            In the case of the call to a "leafref" statement, one must be able to specify the path of a leaf, even through a module passed as an argument to a "full-include" statement.
-            The paths of the nodes contained in the module (refered in the following paragraph as "the included nodes") used in a "full-include" statement are computed as follows.
-            TODO: explain how the path are computed, which basically boils down to concatenating the main module path with the included one ?
+            We present in this section the YANG module defining the "full-include" extension.
+            The module in itself defines solely the 'include' extension.
+            A module importing this extension SHOULD the prefix 'full', so that the statement reads "full:include" when used in the code.
```

Comment:
SHOULD use ?

Reply:
Thanks, abd4d5291b8d0a44c2c708490a1e2426232da841